clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -362,11 +395,19 @@ def open(
         :param kwargs: Extra args
         :return A file-like object from the filesystem
         """
-        context: FilesetContext = self._get_fileset_context(path)
-        return context.get_fs().open(
-            self._strip_storage_protocol(
-                context.get_storage_type(), context.get_actual_path()
-            ),
+        if mode.startswith("w"):
```

Comment:
What's the format of the mode? It could be `rw`?

Reply:
The mode supports: rb(read), wb(write), ab(append). The `b` is optional, which means binary mode. See more details in Python `builtins.open()` method.

Reply:
Should we give a mode a type instead of string?

Reply:
> Should we give a mode a type instead of string?

I think it may not be necessary because the fsspec `open` interface definition also uses strings for `mode`, and the `open` method definition of the Python `builtins` library also uses strings for `mode`. If it is defined as enum type here, it may bring additional understanding costs to users.

Reply:
Does `buitins.open` internal implement use `startswith` to judge the mode?

Reply:
No, Python will not judge the mode, it will pass to the operate system directly, it will judge by the operate system. Here we just judge the mode using `startswith` for auditing.

Reply:
> Maybe we can compare the string value directly like `mode == "w" || mode == "wb"`.

OK, I will update this in this PR: https://github.com/apache/gravitino/pull/5026